src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -140,39 +117,32 @@ public static Logger getLogger() {
 
   public static void gameOverClearCache() {
     // Is static, set to null so that we don't keep the data around after a game is exited
-    s_battleCalculator.setGameData(null);
+    ProBattleUtils.setData(null);
     ProLogUI.clearCachedInstances();
   }
 
-  public IOddsCalculator getCalc() {
-    return s_battleCalculator;
-  }
-
-  @Override
-  public final GameData getGameData() {
-    if (data != null) {
-      return data;
-    } else {
-      return super.getGameData();
-    }
-  }
-
   @Override
   public void stopGame() {
     super.stopGame(); // absolutely MUST call super.stopGame() first
-    s_battleCalculator.cancel(); // cancel any current calcing
+    ProBattleUtils.cancel(); // cancel any current calcing
```

Comment:
A little nit picky. I think cancel is probably good and I can remove this comment but not a huge issue.